tests/unit/scripts/test_update_todos.py

```diff
@@ -33,6 +33,34 @@ def setup_environment():
     os.environ["GITHUB_SHA"] = ""
 
 
+def test_no_todo_flag(tmp_path):
```

Comment:
I think we're still missing a `# notodo` in this file to exclude it right?

Reply:
good point, fixed